builtin/credential/aws-ec2/path_config_sts.go

```diff
@@ -0,0 +1,243 @@
+package awsec2
+
+import (
+	"fmt"
+
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+type awsStsEntry struct {
+	StsRole string `json:"sts_role" structs:"sts_role" mapstructure:"sts_role"`
```

Comment:
That's a good question.

Personally I can't see a good reason to allow multiple STS roles per account. On which criteria would one be chosen over another? The only one I can think of is the region, and I can't see what the benefit would be of maintaining a separate role per region - the Vault server would still need to have permissions to assume them all, for instance, so I don't think we'd gain much security-wise.

I think it feels cleaner to have a single role per account, and let the administrator of that account decide which permissions it has (or doesn't).